aws/resource_aws_api_gateway_rest_api.go

```diff
@@ -221,14 +235,14 @@ func resourceAwsApiGatewayRestApiRead(d *schema.ResourceData, meta interface{})
 
 	d.Set("binary_media_types", api.BinaryMediaTypes)
 
-	arn := arn.ARN{
+	execution_arn := arn.ARN{
```

Comment:
Nit: Standard Go language conventions are to use lowercase [mixed caps for variable naming](https://golang.org/doc/effective_go.html#mixed-caps).